src/Uno.UI.Runtime.Skia.Gtk/GTK/UnoDrawingArea.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using SkiaSharp;
+using WUX = Windows.UI.Xaml;
+
+namespace Uno.UI.Runtime.Skia
+{
+	internal class UnoDrawingArea : Gtk.DrawingArea
+	{
+		private SKBitmap bitmap;
+		private int renderCount;
+		private int InvalidateRenderCount;
+
+		public UnoDrawingArea()
+		{
+			WUX.Window.Current.InvalidateRender
+				+= () =>
+				{
+					QueueDrawArea(0, 0, 10000, 1000);
+				};
+		}
+
+		protected override bool OnDrawn(Cairo.Context cr)
+		{
+			int width, height;
+
+			Console.WriteLine($"Render {renderCount++}");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)